sidekick_core/lib/src/update_sidekick_cli.dart

```diff
@@ -32,13 +33,25 @@ Future<void> main(List<String> args) async {
         // Always execute template updates
         UpdateToolsMigration(targetSidekickCoreVersion),
         UpdateEntryPointMigration(targetSidekickCoreVersion),
+        // Migration steps from git patches
+        ...patchMigrations,
       ],
       onMigrationStepStart: (context) {
         print(' - ${context.step.name}');
       },
       onMigrationStepError: (context) {
         printerr('Migration failed: ${context.step.name}');
         printerr(context.exception?.toString());
+
+        // Migrations from git patches are likely to fail (e.g. if the user
+        // already modified their sidekick CLI)
+        // Therefore git patch migrations are skipped by default,
+        // the printed error contains instructions to check the patch and
+        // apply it manually if necessary
+        if (context.step is GitPatchMigrationStep) {
+          return MigrationErrorHandling.skip;
+        }
```

Comment:
We should print a meaningful error in that case

Reply:
The meaningful error is thrown here: https://github.com/phntmxyz/sidekick/pull/157/files#diff-ae0248a32efccab0fe0a83493d63bef6b5eb793f7fdc74adb171b64d6b0c772dR104 and then printed here: https://github.com/phntmxyz/sidekick/pull/157/files#diff-838c915a049dafb0a8c54aed6c035a94a7383f2720135c9f2e51f78ac515b7c5R44
It was easier to test it this way

Reply:
but the PR link is not printed

Reply:
It prints the link for me:
```
❯ ./sk sidekick update
...
 - Update /tool directory
 - Update entrypoint
 - Fix usage message (1/2) (https://github.com/phntmxyz/sidekick/pull/157)
error: patch failed: sk_sidekick/lib/sk_sidekick.dart:2
error: sk_sidekick/lib/sk_sidekick.dart: patch does not apply
Migration failed: Fix usage message (1/2) (https://github.com/phntmxyz/sidekick/pull/157)
Couldn't apply git patch /var/folders/c0/vw17q1tx6k50xkmzl4wx1pbh0000gn/T/pQqP4A/fix_usage_message_(1_2).sidekick.patch for migration step Fix usage message (1/2) (https://github.com/phntmxyz/sidekick/pull/157).
Check https://github.com/phntmxyz/sidekick/pull/157 for further information.
Try applying the patch manually if necessary.
The patch content is:
...
```

Reply:
Missed that 😉